doc/API.md

```diff
@@ -819,9 +819,15 @@ Response: standard.
 
 #### /renter/hosts/active [GET]
 
-Function: Lists all of the active hosts known to the renter.
+Function: Lists all of the active hosts known to the renter, sorted by
+preference.
 
-Parameters: none
+Parameters:
+```
+numhosts uint64 // Optional
```

Comment:
It's more consistent in the new architecture that @VoidingWarranties and I discussed offline a few days ago. This is mostly a placeholder until we switch to that format.

The new doc architecture clearly marks optional parameters.